im.pidgin.Pidgin.json

```diff
@@ -0,0 +1,134 @@
+{
+  "id": "im.pidgin.Pidgin",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "sdk": "org.gnome.Sdk",
+  "command": "pidgin",
+  "rename-desktop-file": "pidgin.desktop",
+  "rename-icon": "pidgin",
+  "rename-appdata-file": "appdata.xml",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--persist=.purple",
+    "--filesystem=xdg-download:rw",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.config/dconf:ro",
+    "--talk-name=ca.desrt.dconf",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+    "--talk-name=org.freedesktop.Notifications"      
+  ],
+  "cleanup": [
+    "/include",
+    "/share/man",
+    "/share/info",
+    "/share/gtk-doc",
+    "/share/aclocal",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "gtkspell",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/gtkspell/files/2.0.16/gtkspell-2.0.16.tar.gz",
+          "sha256": "8fc7dc560167b2cb7193e76aca625a152dc19b0ebf49816b78539cbb90d80d02"
+        }
+      ]
+    },
+    {
+      "name": "pidgin",
+      "buildsystem": "autotools",
+      "config-opts": ["--disable-vv", "--disable-meanwhile", "--disable-avahi", "--disable-nm", "--disable-tk"],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/pidgin/files/Pidgin/2.13.0/pidgin-2.13.0.tar.bz2",
+          "sha256": "2747150c6f711146bddd333c496870bfd55058bab22ffb7e4eb784018ec46d8f"
+        },
+        {
+          "type": "file",
+          "path": "appdata.xml"
+        }
+      ],
+      "post-install": [
+        "chmod 755 /app/lib/pidgin/perl/auto/Pidgin/Pidgin.so",
+        "chmod 755 /app/lib/purple-2/perl/auto/Purple/Purple.so",
+        "install -Dm644 appdata.xml /app/share/appdata/appdata.xml"
+      ]
+    },
+    {
+      "name": "libgpg-error",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "ftp://ftp.gnupg.org/gcrypt/libgpg-error/libgpg-error-1.32.tar.bz2",
+          "sha256": "c345c5e73cc2332f8d50db84a2280abfb1d8f6d4f1858b9daa30404db44540ca"
+        }
+      ]
+    },
+    {
+      "name": "libgcrypt",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.3.tar.bz2",
+          "sha256": "66ec90be036747602f2b48f98312361a9180c97c68a690a5f376fa0f67d0af7c"
+        }
+      ]
+    },
+    {
+      "name": "libotr",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://otr.cypherpunks.ca/libotr-4.1.1.tar.gz",
+          "sha256": "8b3b182424251067a952fb4e6c7b95a21e644fbb27fbd5f8af2b2ed87ca419f5"
+        }
+      ]
+    },
+    {
+      "name": "pidgin-otr",
```

Comment:
We could put these in their own package. Unsure if its worth it. If upstream gets involved it would make sense to not bundle these.

Reply:
Is there an example of putting them in a seperate package?